cvat-core/src/server-proxy.js

```diff
@@ -499,6 +499,47 @@
                 };
             }
 
+            async function importDataset(id, format, file, callback) {
+                const { backendAPI } = config;
+                const url = `${backendAPI}/projects/${id}/dataset`;
+
+                const formData = new FormData();
+                formData.append('dataset_file', file);
+
+                return new Promise((resolve, reject) => {
+                    async function request() {
+                        try {
+                            const response = await Axios.get(`${url}?action=import_status`, {
+                                proxy: config.proxy,
+                            });
+                            if (response.status === 202) {
+                                if (typeof callback === 'function') callback(response.data);
```

Comment:
```suggestion
                                if (callback) callback(response.data);
```

we already know, that it is a function or null